app/scripts/formValidation.js

```diff
@@ -0,0 +1,29 @@
+const contact = document.querySelector('#contact');
+const fullnameErr = document.querySelector('#fullname-err');
+const emailErr = document.querySelector('#email-err');
+const msgErr = document.querySelector('#msg-err');
+
+contact.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const elements = e.target;
+  const fullname = elements[0].value;
+  const email = elements[1].value;
+  const message = elements[2].value;
+  const fullnameCount = fullname.split(' ').length;
+  if (fullnameCount < 2) {
+    fullnameErr.innerText = 'Please provide your full name!';
+  } else {
+    fullnameErr.innerText = '';
+  }
+  const lwEmail = email.toLowerCase();
+  if (email !== lwEmail) {
+    emailErr.innerText = 'Please provide your valid email address!';
```

Comment:
- [ ] The email message is expected to be descriptive so one knows they must enter an email in lowercase letters only. Kindly modify the error message to be - Please provide a lowercase email address! 👍🏽
- [ ] You could make the color of this error message be red or warning color so it draws attention to the user 👍🏽